dynamo/plot/topography.py

```diff
@@ -265,7 +265,10 @@ def plot_nullclines(
 
                 NCx, NCy = vecfld2d.NCx, vecfld.NCy
     else:
-        NCx, NCy = vecfld_dict["nullcline"][0], vecfld_dict["nullcline"][1]
+        NCx, NCy = (
+            [vecfld_dict["NCx"][index] for index in vecfld_dict["NCx"]],
+            [vecfld_dict["NCy"][index] for index in vecfld_dict["NCy"]],
```

Comment:
what will be the behaviors for this? will this reorder the x/y coordinates of the nullclines? 
            [vecfld_dict["NCx"][index] for index in vecfld_dict["NCx"]],
            [vecfld_dict["NCy"][index] for index in vecfld_dict["NCy"]],

Reply:
This will read the nullclines from the dictionary to form a list of x and y coordinates. The order should be the same. [Here](https://realpython.com/python-ordereddict/#:~:text=Since%20Python%203.6%2C%20regular%20dictionaries,inserted%20into%20the%20underlying%20dictionary.) is a reference indicating regular dictionaries have kept their items in the same order that they were inserted into the underlying dictionary since python 3.6.